### ISSUE TITLE
Obscure log entries when editing a flow with a trigger

### ISSUE THREAD

**Issue by aku** [kestra-io/kestra#3969]
### Describe the issue

I have a flow with a Kafka trigger.
When I edit the flow Kestra produces log entires with following content:

<img width="1391" alt="image" src="https://github.com/kestra-io/kestra/assets/678596/741ba3ac-2a91-433f-bcb1-7b945a054de0">

I guess "null" and "InterruptExceptions" are not very descriptive. Also, it might lead to conclusion that something is broken.

### Environment

- Kestra Version: 0.17.1
- Operating System (OS/Docker/Kubernetes): ubuntu 12
- Java Version (if you don't run kestra in Docker): jdk22

**Comment by loicmathieu** [comment]
Hi,
Thanks for opening the issue.
Can you confirm it is when using a realtime trigger?

If so, this is a duplicate of https://github.com/kestra-io/kestra/issues/3882, we already start working on it.

**Comment by aku** [comment]
> Hi, Thanks for opening the issue. Can you confirm it is when using a realtime trigger?
> 
> If so, this is a duplicate of #3882, we already start working on it.

I use a regular "io.kestra.plugin.kafka.Trigger" trigger with short interval for now.

```yaml
triggers:
  - id: read_orders_from_kafka
    type: io.kestra.plugin.kafka.Trigger
    keyDeserializer: STRING
    valueDeserializer: JSON
    interval: PT5S
    topic: [REDACTED]
    groupId: [REDACTED]
    maxRecords: 7000
    properties: [REDACTED]
```

**Comment by loicmathieu** [comment]
It's definitely an issue with real-time triggers, but for triggers, it happens if Kestra is stopped and a trigger is still being evaluated.

**Comment by aku** [comment]
> It's definitely an issue with real-time triggers, but for triggers, it happens if Kestra is stopped and a trigger is still being evaluated.

I've seen this behaviour in 2 cases - either when I edit a flow while trigger is active or when I restart worker components.
Anyway, it would be nice to either hide these messages or change them to something more descriptive.
Currently, it is not clear what happens if I update a flow  with a grigger - will it kill an active execution by accident or it is nothing to worry about.

**Comment by loicmathieu** [comment]
> Anyway, it would be nice to either hide these messages or change them to something more descriptive.

We could have a look when we fix the corresponding issue for realtime trigger.

> Currently, it is not clear what happens if I update a flow with a grigger - will it kill an active execution by accident or it is nothing to worry about.

No, it will use this trigger for the next evaluation, so the next execution will be based on the new version of the trigger.

**Comment by aku** [comment]
> No, it will use this trigger for the next evaluation, so the next execution will be based on the new version of the trigger.

Actually, there might be a problem with "new version". In my case a new revision got picked only after I manually disabled\enabled the whole flow. https://github.com/kestra-io/kestra/issues/3970

**Comment by loicmathieu** [comment]
> Actually, there might be a problem with "new version". In my case a new revision got picked only after I manually disabled\enabled the whole flow. https://github.com/kestra-io/kestra/issues/3970

Seems like a bug, and already tracked in https://github.com/kestra-io/kestra/issues/3970
